src/running/runConfig.ts

```diff
@@ -38,7 +38,7 @@ export async function runConfig(
 					}),
 				),
 			),
-			rules: use.rules.flat() as ConfigRuleDefinition[],
+			rules: (use.rules?.flat() ?? []) as ConfigRuleDefinition[],
```

Comment:
🐛 `use` objects should allow omitting `rules`. This PR is an example of why you'd want to: to just run the formatter on them.